### PR TITLE
Updated readme docs

### PR DESCRIPTION
**Pull request by saratchandrakarumuri**
* Updated api entities according to usage in swift 3.0
* Updated default values

### CODE REVIEW

File: arcgis-runtime-samples-macos/Maps/Manage operational layers/README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ The map in the sample app comes with two map image layers already added. The lis
 
 ##How it works
 
-`AGSMap` has a property called `operationalLayers` which is a list of `AGSLayer`. To add a layer the app uses the `(void) - addObject:` method on the list (`AGSList`). To remove a layer it uses `(void) - removeObjectAtIndex:` method and for re-ordering it uses a combination of those methods.
+`AGSMap` has a property called `operationalLayers` which is a list of `AGSLayer`. To add a layer the app uses the `add() -> void` method on the list (`AGSList`). To remove a layer it uses `removeObject() -> void` method and for re-ordering it uses a combination of those methods.
```
Comment (singhgag) [inline arcgis-runtime-samples-macos/Maps/Manage operational layers/README.md]: @saratchandrakarumuri why are you doing `add() -> void`. Just do `add()` instead.

Comment (singhgag) [inline arcgis-runtime-samples-macos/Maps/Manage operational layers/README.md]: For `removeObject() -> Void` it should be `removeObject(at:)`

File: arcgis-runtime-samples-macos/Maps/Map rotation/README.md [hunk 2]
```diff
@@ -7,9 +7,9 @@ This sample demonstrates how to rotate a map using a slider control.
 
 ##How it works
 
-To set rotation of a map, `AGSMapView` provides a few methods. You can use the `setViewpointRotation:completion:` method and give it an angle in degrees. Or you can create a rotated `AGSViewpoint` using `initWithCenter:scale:rotation:` or `initWithTargetExtent:rotation:` and set it using the following methods on `AGSMapView` - `setViewpoint:`, `setViewpoint:completion:`, `setViewpoint:duration:completion:`, `setViewpoint:duration:curve:completion:`.
+To set rotation of a map, `AGSMapView` provides a few methods. You can use the `setViewpointRotation(_:completion:)` method and give it an angle in degrees. Or you can create a rotated `AGSViewpoint` using `init(center:scale:rotation:)` or `init(targetExtent:rotation:` and set it using the following methods on `AGSMapView` - `setViewpoint(_:)`, `setViewpoint(_:completion:)`, `setViewpoint(_:duration:completion:)`, `setViewpoint(_:duration:curve:completion:)`.
```
Comment (singhgag) [inline arcgis-runtime-samples-macos/Maps/Map rotation/README.md]: `init(targetExtent:rotation:` is missing a closing bracket `)`

File: arcgis-runtime-samples-macos/Maps/Set initial map area/README.md [hunk 3]
```diff
@@ -4,7 +4,7 @@ This sample displays a map at a specific viewpoint defined by an envelope.
 
 ##How it works
 
-In this sample a viewpoint (AGSViewpoint) is constructed from an envelope (AGSEnvelope). Before the map is loaded the AGSMap's' initialViewpoint is set to this viewpoint. When you assign the map to the AGSMapview the map starts to load and the map zooms to this initial area.
+In this sample a viewpoint (`AGSViewpoint`) is constructed from an envelope (`AGSEnvelope`). Before the map is loaded the AGSMap's' initialViewpoint is set to this viewpoint. When you assign the map to the `AGSMapview` the map starts to load and the map zooms to this initial area.
```
Comment (singhgag) [inline arcgis-runtime-samples-macos/Maps/Set initial map area/README.md]: Remove the brackets around class names. And add ` where ever necessary.

File: arcgis-runtime-samples-macos/Maps/Set map spatial reference/README.md [hunk 4]
```diff
@@ -6,7 +6,7 @@ This sample demonstrates how you can set the spatial reference on a `AGSMap` and
 
 ##How it works
 
-The map in the sample app is initialized with `esriSRProjCS_World_Bonne (WKID: 54024)` spatial reference using `(instancetype) - initWithSpatialReference:` initializer. Hence the map image layer that is added, with default spatial reference as `GCS_WGS_1984 (WKID: 4326)`, gets re-projected to map's spatial reference.
+The map in the sample app is initialized with `esriSRProjCS_World_Bonne (WKID: 54024)` spatial reference using `(instancetype) - init(spatialReference:)` initializer. Hence the map image layer that is added, with default spatial reference as `GCS_WGS_1984 (WKID: 4326)`, gets re-projected to map's spatial reference.
```
Comment (singhgag) [inline arcgis-runtime-samples-macos/Maps/Set map spatial reference/README.md]: Remove the `(instancetype) - ` from the init

File: arcgis-runtime-samples-macos/Maps/Show magnifier/README.md [hunk 5]
```diff
@@ -6,7 +6,7 @@ This sample demonstrates how you can tap and hold on a map to get the magnifier.
 
 ##How it works
 
-`AGSMapView` has a property called `magnifierEnabled` that indicates whether a magnifier should be shown on the map when the user performs a tap and hold gesture. Its default value is `NO` or `false`. You can also use the `allowMagnifierToPanMap` property to indicate whether the map should be panned automatically when the magnifer gets near the edge of the map's bounds.
+`AGSMapView` has a property called `isMagnifierEnabled` that indicates whether a magnifier should be shown on the map when the user performs a tap and hold gesture. Its default value is `YES` or `true`. You can also use the `allowMagnifierToPan` property to indicate whether the map should be panned automatically when the magnifer gets near the edge of the map's bounds.
```
Comment (singhgag) [inline arcgis-runtime-samples-macos/Maps/Show magnifier/README.md]: `isMagnifierEnabled` is now on `interactionOptions`. We should make that change.

**Comment by saratchandrakarumuri** [comment]
@singhgag Updated the docs with suggested changes

**Pull request merged**